services/api/README.md

```diff
@@ -138,7 +138,31 @@ router
 
 ## Updating E-Mail Templates
 
-E-mail templates can be found in `emails/src`. When changes are made, run the following command to optimize the emails for mail readers:
+E-mail templates can be found in `emails`.
+There is a layout.html that contains the styling and default layout, and then we got template for each email, that gets injected into the layout.
```

Comment:
nit: "and then we got template for each email" -> "and a template for each email"
just kind of a tone thing